generate.py

```diff
@@ -65,6 +80,9 @@ def _validate_args(args):
 
     if args.frame_num is None:
         args.frame_num = cfg.frame_num
+    
+    if args.sample_solver != "euler":
```

Comment:
Add an extra `use_lora` check to avoid affecting the original inference process?

```python
if use_lora and args.sample_solver != "euler":
    pass
```